processing/src/main/java/org/apache/druid/data/input/impl/JsonInputFormat.java

```diff
@@ -156,6 +158,16 @@ public InputEntityReader createReader(InputRowSchema inputRowSchema, InputEntity
     }
   }
 
+  @JsonIgnore
+  @Override
+  public long getWeightedSize(@Nullable CompressionUtils.Format compressionFormat, long size)
```

Comment:
I see this implemented for json, csv, and parquet. Please implement it for Avro, ORC, regex, and delimited as well. Delimited and regex can be the same as CSV. Avro and ORC can be the same as Parquet.

Reply:
good call, added for those as well.